### PR TITLE
Onboarding content audit 4 – data design quickstart

### PR DESCRIPTION
**Pull request by thoragudf**
Updated the data design quickstart guide quite a bit.

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝 Changed routes:


| [**/data-design/quick-start**](https://docs-3g35un0u5.avo-dev.app/data-design/quick-start) |
|:-:|
|<a href="https://docs-3g35un0u5.avo-dev.app/data-design/quick-start">Screenshot of /data-design/quick-start</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/SLicLR2lANMLS-9Yts0a6-yBsjs4nHR8_T7AdwfpPlLqQCOKz7E4rItebbz-qy-pTLkiN9FCJL0sS1Yq1500-w.png">(view full size)</a> |







Commit 2e4e864251d3ee3d232b652a35344aba1f137352 (https://docs-3g35un0u5.avo-dev.app).

File: pages/data-design/quick-start.mdx [hunk 1]
```diff
@@ -12,93 +13,168 @@ When you have your tracking plan defined in Avo you can check out our [code gene
...
+- [Build your tracking plan from scratch](/data-design/start-data-design)
 
-![Import button in Avo workspace](/images/audit/import-button.png)
-
-You can import a tracking plan from any of the following sources: Excel or Google Sheets (.csv), Mixpanel Lexicon or Amplitude Data. Learn more about the available import formats in the [importing docs](/publishing/importing). If you have your tracking plan on a format that's currently not supported by the tracking plan importer, [reach out](/publishing/importing) to us and we'll import it for you.
+<PageLink
+  image={'/docs/images/svg/analyticsmanager.svg'}
+  title="Get your tracking plan into Avo"
+  description="An overview of how you can get your existing tracking plan into Avo."
+  href={'/docs/audit/get-tracking-plan-into-avo'}
```
Comment (hugihlynsson) [inline pages/data-design/quick-start.mdx:33]: This link did not resolve for me...

**Review by hugihlynsson — APPROVED** [review]
One 404 link, otherwise solid updates. Love the new videos, they're so smooth!

File: pages/data-design/quick-start.mdx (hunk 1 above)
Comment (thoragudf) [reply pages/data-design/quick-start.mdx]: ooops

**Pull request merged**